app/javascript/packs/polyfills.js

```diff
@@ -0,0 +1,2 @@
+import 'babel-polyfill'
+import 'whatwg-fetch'
```

Comment:
This is needed to make fetch and promises work in IE11.
I've found (in this [GH Issue](https://github.com/rails/webpacker/issues/1066)) this is the recommended way to do it when using webpacker.
Do you know a better way @didierofrivia @josemigallas ?

Reply:
You've used it differently in the past: https://github.com/3scale/porta/blob/master/app/javascript/src/Stats/inlinechart/index.jsx#L6. However, if this new one is a preferred way by webpackers, I'd just keep it consistent.

Reply:
I will keep it consistent with inlinechart component. Later, when we update webpacker gem, we can change it if we decide that.